style.css

```diff
@@ -429,3 +593,133 @@ form {
   position: relative;
   left: 100px;
 }
+
+@media screen and (min-width: 768px) {
+  .img-bg {
+    background-image: url("images/desktopimage.png");
+    background-position: right top;
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin: 50px 8px 76px 34px;
+  }
+
+  .container {
+    padding: 0 70px;
+  }
+
+  .bio {
+    padding-top: 170px;
+    position: relative;
+    right: 30px;
+  }
+
+  .project-container1 {
+    display: flex;
+    flex-direction: row;
+    gap: 46px;
+    margin-top: 0;
+  }
+
+  .project-container2 {
+    padding: 24px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    border-radius: 24px;
+    gap: 46px;
+  }
+
+  .project-container3 {
+    border-radius: 24px;
+    padding: 24px;
+    display: flex;
+    flex-direction: row;
+    gap: 46px;
+  }
+
+  .project-container4 {
+    border-radius: 24px;
+    padding: 24px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    gap: 10px;
+  }
+
+  .project-main-wrapper1 {
+    padding: 114px 142px;
+    gap: 142px;
+  }
+
+  .paragraph {
+    margin-right: 100px;
+  }
+
+  .containertwo {
+    display: flex;
+    flex-direction: row;
+    padding: 142px;
+    gap: 24px;
+  }
+
+  .project-main-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+  }
+
+  .social-media {
+    display: flex;
+    flex-direction: column;
+    margin-left: 80px;
+  }
+
+  .social-links2 {
+    padding: 0;
+  }
+
+  .langList {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 12px;
+  }
+
+  .contact h2 {
+    margin: 130px 319px 0 320px;
+  }
+
+  .contact p {
+    margin: 12px 435px 0 436px;
+  }
+
+  #text {
+    margin: 12px 497px 0 496px;
+    border-radius: 0;
+  }
+
+  #email {
+    margin: 12px 497px 0 496px;
+    border-radius: 0;
+  }
+
+  #message {
+    margin: 12px 497px 0 496px;
+    border-radius: 0;
+  }
+
+  .form-btn {
+    margin: 0 0 139px 649px;
+  }
+
+  .project-review {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .contact-hr {
+    display: none;
+  }
+}
```

Comment:
- [ ] As stated the project requirement You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] For desktop version of your Portfolio for "Top Navigation" section The hamburger button is gone (shouldn't be displayed in this screen size), The Item Navigation should be aligned to the right and The Logo should be aligned to the left just like the Figma design.

Your  "Top Navigation" section Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/199928835-cfede617-b84f-4833-bb6c-b733b19ad24b.PNG)

The Figma design  "Top Navigation" section Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/199929277-dd512d41-3439-4f1b-a088-f7563368eca5.PNG)

- [ ] For "contact me" section , The elements of the form  should  centered on the page including the "Get in touch button". Kindly adjust it.

Your  "contact me"  section Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/199929969-cb5f237f-1f67-4478-aa29-a1a7a0012a56.PNG)

Your  "contact me"  section Look likes this : for screen size of 1080px is not aligned centered, Kindly adjust it.

![Capture](https://user-images.githubusercontent.com/93598189/199930383-d9bfabbc-b110-431c-810c-83b44a6163ca.PNG)


The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/199930710-8a2fc07b-5f28-4521-aa4b-4a5a256cab58.PNG)